angular-ngrx-scss/src/app/organization/services/organization.service.ts

```diff
@@ -0,0 +1,41 @@
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { environment } from 'src/environments/environment';
+import {
+  OrganizationReposApiParams,
+  OrganizationReposApiResponse,
+} from './organization.interfaces';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class OrganizationService {
+  constructor(private http: HttpClient) {}
+
+  getOrganizationRepos(
+    organizationName: string,
+    params?: OrganizationReposApiParams,
+  ): Observable<OrganizationReposApiResponse> {
+    const defaultParams = {
+      type: 'all',
+      sort: 'updated',
+      direction: 'desc',
+      per_page: 30,
+      page: 1,
+    };
+
+    const url = `${environment.githubUrl}/orgs/${encodeURIComponent(
+      organizationName,
```

Comment:
question: Is this comma needed? If not I'd recommend removing it.

Reply:
I just confirmed prettier enforces this.